docs/CHANGELOG.rst

```diff
@@ -24,6 +33,12 @@ Added
 -----
 - Support Python 3.7
 
+Fixed
+-----
+- Fix handling of no-adapters input in workflows
```

Comment:
Workflow fix is in a commit without changes in tests. Additional tests for no-adapters case appear in another commit.